# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/cinepaint/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./plug-ins/psd/psd.c.orig    2004-02-07 00:20:23.000000000 +0100
+++ ./plug-ins/psd/psd.c       2004-07-10 14:32:37.772325057 +0200
@@ -299,7 +299,7 @@
                           gint    *nreturn_vals,
                           GimpParam **return_vals);
 
-static GImageType  psd_type_to_gimp_type      (psd_imagetype  psdtype);
+static GDrawableType  psd_type_to_gimp_type   (psd_imagetype  psdtype);
 static GImageType  psd_type_to_gimp_base_type (psd_imagetype  psdtype);
 static GLayerMode  psd_lmode_to_gimp_lmode    (gchar          modekey[4]);
 static GimpUnit    psd_unit_to_gimp_unit      (gint           psdunit);
